packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx

```diff
@@ -109,32 +132,61 @@ const AgendaItem = (props: Props) => {
     ref.current && ref.current.scrollIntoView({behavior: 'smooth'})
   }, [])
 
+  const handleIconClick = (e: React.MouseEvent) => {
+    e.stopPropagation()
+    UpdateAgendaItemMutation(
+      atmosphere,
+      {updatedAgendaItem: {id: agendaItemId, pinned: !pinned}},
+      {meetingId}
+    )
+  }
+
   const handleRemove = () => {
     RemoveAgendaItemMutation(atmosphere, {agendaItemId})
   }
+
+  const getIcon = () => {
+    if (pinned && isHovering) return <SvgIcon alt='unpinIcon' src={unpinIcon} />
+    else if (!pinned && !isHovering) return <Avatar hasBadge={false} picture={picture} size={24} />
+    else return <SvgIcon alt='pinnedIcon' src={pinIcon} />
+  }
+
   return (
-    <AgendaItemStyles title={content}>
-      <MeetingSubnavItem
-        label={content}
-        metaContent={
-          <AvatarBlock>
-            <Avatar hasBadge={false} picture={picture} size={24} />
-          </AvatarBlock>
-        }
-        isDisabled={isDisabled}
-        onClick={onClick}
-        isActive={isActive}
-        isComplete={isComplete}
-        isDragging={isDragging}
-        isUnsyncedFacilitatorStage={isUnsyncedFacilitatorStage}
-      />
-      <DeleteIconButton
-        aria-label={'Remove this agenda topic'}
-        icon='cancel'
-        onClick={handleRemove}
-        palette='midGray'
-      />
-    </AgendaItemStyles>
+    <>
+      <AgendaItemStyles
+        onMouseEnter={() => setIsHovering(true)}
```

Comment:
consider onMouseOver instead of onMouseEnter for this edge case:
1. on a team dashboard, have 1 agenda item
2. put your mouse cursor over the agenda item input and click
3. enter the agenda item & hit enter
4. notice that the cursor is now _over_ the 2nd agenda item, but isHovering is false

Reply:
Good point! Thanks Matt. I've pushed the update to my branch but I see you beat me to it. Great to see that it's now in master 🎉